Configuration/Settings.yaml

```diff
@@ -1,3 +1,24 @@
+Flowpack:
+  ElasticSearch:
+    ContentRepositoryAdaptor:
+      driver:
+        version: '2.x'
+        mapping:
+          '1.x':
+            query: 'Flowpack\ElasticSearch\ContentRepositoryAdaptor\Driver\Version1\Query\FilteredQuery'
```

Comment:
Currently the most breaking change, before this one is configured in Objects.yaml ...